packages/app/src/ExplorerTabs.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from 'react';
+import { FiList, FiSearch } from 'react-icons/fi';
+
+import styles from './ExplorerTabs.module.css';
+import Explorer from './explorer/Explorer';
+import { useDataContext } from './providers/DataProvider';
+import SearchBoxContainer from './search/SearchTabContainer';
+
+interface Props {
+  selectedPath: string;
+  onSelect: (path: string) => void;
+}
+
+enum Tab {
+  Explore = 'explore',
+  Search = 'search',
+}
+
+function ExplorerTabs(props: Props) {
+  const { selectedPath, onSelect } = props;
+
+  const [tab, setTab] = useState<Tab>(Tab.Explore);
+
+  const [searchValue, setSearchValue] = useState<string>('');
```

Comment:
The state storing the `value` of the search `input` is handled here so that it can be restored when tabbing from the search tab to the explorer tab and back to the search tab.